cluster/common.sh

```diff
@@ -38,6 +38,10 @@ KUBE_RELEASE_VERSION_REGEX="^v(0|[1-9][0-9]*)\\.(0|[1-9][0-9]*)\\.(0|[1-9][0-9]*
 # kube::release::parse_and_validate_ci_version()
 KUBE_CI_VERSION_REGEX="^v(0|[1-9][0-9]*)\\.(0|[1-9][0-9]*)\\.(0|[1-9][0-9]*)-(beta|alpha)\\.(0|[1-9][0-9]*)(\\.(0|[1-9][0-9]*)\\+[-0-9a-z]*)?$"
 
+python="python"
+if [[ "$($python --version 2>&1)" != "Python 2."* ]];then
+    python="python2"
```

Comment:
I think that got introduced in a commit I rebased onto recently. I remember distinctly having to symlink python to python2 :)